index.html

```diff
@@ -37,6 +37,48 @@
     </nav>
 
 
+    <main>
+        <div class="theme">
+            <div class="flex">
+                <div>
+                    <h1 class="h1"> Meow in! </h1>
+                    <p> You know that feeling when you miss sunshine?
+                        Yep, <em> that </em> is how much we missed you. <strong> Welcome back</strong>.
+                    </p>
+                </div>
+                <img src="https://www.smashingmagazine.com/images/smashing-cat/singing-barista.svg" alt="logo"
+                    width="250" height="100" class="floatright" />
+            </div>
+            <form class="main-form">
+                <label class="block">
+                    <span class="margin-bottom"> Your email </span>
+                    <input class="login-input" type="text" name="email" required>
+                </label>
+
+                <label>
+                    <span class="margin-bottom margin-top"> Memorable password (yep, that one) </span>
+                    <div class="flex margin-bottom margin-top">
+                        <input id="passwordInput" type="password" name="password" class="login-input--border-left"
+                            required>
+                        <button type="button" class="showbutton"> <svg width="10" height="10"
```

Comment:
This class shouldn't be tied to what the text inside is. What if we replace the text from `Show` to `Hide`? Will we change the class as well, to `hidebutton`?

Of course not. The class name of the button should be independent of the content. This one here could be `button-orange` and the other one, could be `button-green`. This way, we have 2 button variants that we can reuse in our app.